tests/update/patch_before_migration.pm

```diff
@@ -0,0 +1,68 @@
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Patch SLE11* and SLE12* qcow2 images before migration (offline)
+# Maintainer: Dumitru Gutu <dgutu@suse.de>
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+use registration;
+
+sub patching_sle() {
+
+    select_console 'root-console';
+
+    set_var("VIDEOMODE",    'text');
+    set_var("SCC_REGISTER", 'installation');
+
+    # stop packagekit service
+    script_run "systemctl mask packagekit.service";
+    script_run "systemctl stop packagekit.service";
+
+    assert_script_run("zypper lr && zypper mr --disable --all");
+    save_screenshot;
+    yast_scc_registration();
+    assert_script_run('zypper lr -d');
+    minimal_patch_system(version_variable => 'HDDVERSION');
```

Comment:
Thank you for taking a look :) I'm adding the variable to control the minimal and full patching.